site/src/Librecores/ProjectRepoBundle/Entity/OrganizationMember.php

```diff
@@ -0,0 +1,204 @@
+     * @ORM\JoinColumn(name="organizationId", referencedColumnName="id", nullable=FALSE)
+     */
+    protected $organization;
+
+    /**
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="organizationMemberships")
+     * @ORM\JoinColumn(name="userId", referencedColumnName="id", nullable=FALSE)
+     */
+    protected $user;
+
+    /**
+     * @var string
+     *
+     * @Assert\Choice(choices = {"REQUEST", "DENY", "SUPPORT", "MEMBER", "ADMIN"})
+     * @ORM\Column(type="string")
+     */
+    protected $permissions = self::PERMISSIONS_REQUEST;
+
+    // metadata
+    /**
+     * When was this mapping created?
+     *
+     * @var \DateTime
+     * @ORM\Column(type="datetime")
+     */
+    protected $createdAt;
+
+    /**
+     * When was this mapping last updated?
+     *
+     * @var \DateTime
+     * @ORM\Column(type="datetime")
+     */
+    protected $updatedAt;
+
+
+    public function __construct()
+    {
+ 
+    }
+
+    /**
+     * Hook on the first storing of this object
+     *
+     * @ORM\PrePersist()
+     */
+    public function prePersist()
+    {
+        $this->createdAt = new \DateTime;
+        $this->updatedAt = new \DateTime;
+    }
+
+    /**
+     * Hook before each update
+     *
+     * @ORM\PreUpdate()
+     */
+    public function preUpdate()
+    {
+        $this->updatedAt = new \DateTime;
+    }
+
+    /**
+     * Get id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set user
+     *
+     * @param User $user
+     * @return OrganizationMember
+     */
+    public function setUser(User $user = null)
+    {
+        if ($this->user !== null) {
+            $this->user->removeOrganizationMembership($this);
+        }
+
+        if ($user !== null) {
+            $user->addOrganizationMembership($this);
+        }
+
+        $this->user = $user;
+
+        return $this;
+    }
+
+    /**
+     * Get user
+     *
+     * @return User
+     */
+    public function getUser()
+    {
+        return $this->user;
+    }
+
+    /**
+    * Set organization
+    *
+    * @param Organization $organization
+    * @return OrganizationMember
+    */
+    public function setOrganization(Organization $organization = null)
+    {
+        if ($this->organization !== null) {
+            $this->organization->removeMember($this);
+        }
+
+        if ($organization !== null) {
+            $organization->addMember($this);
+        }
+
+        $this->organization = $organization;
+
+        return $this;
+    }
+
+    /**
+     * Get organization
+     *
+     * @return Organization
+     */
+    public function getOrganization()
+    {
+        return $this->organization;
+    }
+
+    /**
+     * Set permissions
+     *
+     * @param string $permissions One of the self::PERMISSIONS_* constants
+     * @throws \InvalidArgumentException
+     */
+    public function setPermissions($permissions)
```

Comment:
setPermissions() suggests that you set an array of permissions, but in fact it's only one. So rename to setPermission()?